### ISSUE TITLE
Can't browser the cluster from web portal after upgrading to v2.6

### ISSUE THREAD

**Issue by hwchiu** [rancher/rancher#34621]
**Rancher Server Setup**
- Rancher version: v2.6.0
- Installation option (Docker install/Helm Chart):
   - Helm: "v3.4.2"
   - Kubernetes: v1.16.2 (RKE1)
- Proxy/Cert Details: Let's Encrypted certificate

**Describe the bug**
After upgrading Rancher from v2.5.5 to v2.6.0, I'm no longer access my custom cluster from the new Rancher portal.

I got the following screen when I click the name of my existing clusters.

![](https://i.imgur.com/fAeYviF.png)

I have tried the following ways but none of them works
1. Clean the cache
2. Use incognito windows

The worst part is the local cluster is gone after the upgrade, I can see three clusters in my previous v2.5.9 version, one for the local cluster(RKE for Rancher), two for my custom clusters, but the local cluster is disappeared from the portal.
![](https://i.imgur.com/VaxxZev.png)

**To Reproduce**
Prepare Rancher v2.5.5 cluster and then use the Helm (v3.4.2) to upgrade it to v2.6.0

 

**Result**

**Expected Result**
1. Can successfully browser existing cluster with new Rancher UI.
**Screenshots**

**Additional context**
Log message from Rancher Pod
```
2021/09/02 16:57:22 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:57:22 [ERROR] error syncing 'local': handler kubernetes-provider: Operation cannot be fulfilled on clusters.management.cattle.io "local": StorageError: invalid object, Code: 4, Key: /registry/management.cattle.io/clus
ters/local, ResourceVersion: 0, AdditionalErrorMsg: Precondition failed: UID in precondition: 5c9e250b-ec68-4911-9a46-3ecabc644c4d, UID in object meta: , handler cluster-watch: clusterregistrationtokens.management.cattle.io "defaul
t-token" is forbidden: unable to create new content in namespace local because it is being terminated, handler mgmt-cluster-remove: Operation cannot be fulfilled on clusters.management.cattle.io "local": StorageError: invalid objec
t, Code: 4, Key: /registry/management.cattle.io/clusters/local, ResourceVersion: 0, AdditionalErrorMsg: Precondition failed: UID in precondition: 5c9e250b-ec68-4911-9a46-3ecabc644c4d, UID in object meta: , requeuing
2021/09/02 16:57:31 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:57:37 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:57:44 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:57:49 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:57:50 [ERROR] error syncing 'fleet-local/local': handler workspace-backport: fleetworkspaces.management.cattle.io "fleet-local" not found, requeuing
2021/09/02 16:57:55 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:57:59 [ERROR] Failed to handling tunnel request from 10.42.171.100:56624: response 400: cluster not found
2021/09/02 16:57:59 [INFO] Active TLS secret serving-cert (ver=310839050) (count 23): map[listener.cattle.io/cn-10.42.171.100:10.42.171.100 listener.cattle.io/cn-10.42.171.80:10.42.171.80 listener.cattle.io/cn-10.42.171.81:10.42.17
1.81 listener.cattle.io/cn-10.42.171.82:10.42.171.82 listener.cattle.io/cn-10.42.171.88:10.42.171.88 listener.cattle.io/cn-10.42.171.89:10.42.171.89 listener.cattle.io/cn-10.42.171.91:10.42.171.91 listener.cattle.io/cn-10.42.238.12
0:10.42.238.120 listener.cattle.io/cn-10.42.238.127:10.42.238.127 listener.cattle.io/cn-10.42.238.67:10.42.238.67 listener.cattle.io/cn-10.42.238.69:10.42.238.69 listener.cattle.io/cn-10.42.238.72:10.42.238.72 listener.cattle.io/cn
-10.42.90.71:10.42.90.71 listener.cattle.io/cn-10.42.90.74:10.42.90.74 listener.cattle.io/cn-10.42.90.78:10.42.90.78 listener.cattle.io/cn-10.42.90.79:10.42.90.79 listener.cattle.io/cn-10.42.90.81:10.42.90.81 listener.cattle.io/cn-
10.42.90.95:10.42.90.95 listener.cattle.io/cn-127.0.0.1:127.0.0.1 listener.cattle.io/cn-localhost:localhost listener.cattle.io/cn-rancher.cattle-system:rancher.cattle-system listener.cattle.io/cn-testing.aether.onlab.us:testing.aet
her.onlab.us listener.cattle.io/fingerprint:SHA1=2DC3E71D4977C36BFB417E0AC4648C6385865D9B listener.cattle.io/hash:a5639694794df315a4c33e4bafc20a2c1e613111e84b586fa75eb2a8f3a4cf4d]
2021/09/02 16:57:59 [INFO] Active TLS secret tls-rancher-internal (ver=310839051) (count 1): map[listener.cattle.io/cn-10.43.150.82:10.43.150.82 listener.cattle.io/fingerprint:SHA1=6124651E78FCBFBF00304CFE0EC22301A08BE7F9]
2021/09/02 16:58:01 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:58:04 [INFO] error in remotedialer server [400]: read tcp 10.42.90.95:443->10.42.171.91:55172: use of closed network connection
W0902 16:58:04.612419      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.RoleBinding ended with: an error on the server ("unable to decode an event from the w
atch stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612462      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.Pod ended with: an error on the server ("unable to decode an event from the watch str
eam: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612463      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.ClusterRole ended with: an error on the server ("unable to decode an event from the w
atch stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612482      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.Endpoints ended with: an error on the server ("unable to decode an event from the wat
ch stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612493      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.ClusterRoleBinding ended with: an error on the server ("unable to decode an event fro
m the watch stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612501      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.Service ended with: an error on the server ("unable to decode an event from the watch
 stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612552      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.Secret ended with: an error on the server ("unable to decode an event from the watch
stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612559      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.Role ended with: an error on the server ("unable to decode an event from the watch st
ream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612593      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.Namespace ended with: an error on the server ("unable to decode an event from the wat
ch stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612599      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.APIService ended with: an error on the server ("unable to decode an event from the wa
tch stream: tunnel disconnect") has prevented the request from succeeding
W0902 16:58:04.612653      32 reflector.go:437] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: watch of *v1.ServiceAccount ended with: an error on the server ("unable to decode an event from th
e watch stream: tunnel disconnect") has prevented the request from succeeding
2021/09/02 16:58:07 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:58:09 [INFO] Handling backend connection request [10.42.171.91]
2021/09/02 16:58:09 [ERROR] Failed to handling tunnel request from 10.42.171.100:56730: response 400: cluster not found
2021/09/02 16:58:12 [ERROR] failed to call leader func: failed to add management data: namespaces "c-8b6tz" not found
2021/09/02 16:58:14 [ERROR] Failed to handling tunnel request from 10.42.238.94:44528: response 400: cluster not found
```

**Comment by dveni** [comment]
Experimenting a similar issue, imported a custom cluster but cannot open it in the explorer (it appears with a gray logo instead of blue at the navbar). Looking at the logs I have several errors like this one:
```
[ERROR] error syncing 'cattle-system': handler resourceQuotaSyncController: projects.management.cattle.io "c-m-6tf7kztp/p-jnxlr" not found, handler resourceQuotaUsedLimitController: projects.management.cattle.io "c-m-6tf7kztp/p-jnxlr" not found, requeuing
```
Referring to cattle-system, kube-system...

**Comment by aquam8** [comment]
I have the same problem. I see all imported clusters, as Available (EKS 1.21), but none are clickable to get to the resources.
If i go to the Cluster Manager, i can click on the downstream cluster and can see the nodes but that's it.
It also shows the Registration tab with the `kubectl apply -f xxx` command just when to register the downstream cluster when it's not registered yet. I tried re-running the command from the downstream cluster, but it's not fixing the problem.
